<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTMLConverter</class>
 <widget class="QWidget" name="HTMLConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTML Converter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" rowspan="4" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="plainText">
      <string>
</string>
     </property>
     <property name="placeholderText">
      <string>HTML result</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="convert_html_btn">
     <property name="text">
      <string>Convert to HTML</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="send_C_btn">
     <property name="text">
      <string>Send C to program</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="path_line_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>/path/to/a/text/file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse_btn">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
